src/server/requestHandlers.js

```diff
@@ -206,7 +252,15 @@ sjrk.storyTelling.server.buildBinaryRenameMap = function (blocks, files) {
     return binaryRenameMap;
 };
 
-// Rotates an image to be oriented based on its EXIF orientation data, if present
+/**
+ * Rotates an image to be oriented based on its EXIF orientation data, if present.
+ * Uses the npm package "jpeg-autorotate": https://www.npmjs.com/package/jpeg-autorotate
+ *
+ * @param {Object} file - an image file to process
+ * @param {Object} options - additional options to pass into the rotate call
+ *
+ * @return {Promise} - a fluid-flavoured promise

```

Comment:
Be more specific with what the returned promise carries. This comment applies to other docs too.